build-steps/release/check-uploaded-artifacts.gradle

```diff
@@ -4,12 +4,13 @@ import java.util.jar.JarFile
 import java.util.jar.Manifest
 
 File scriptRoot = currentScriptRootOf this
+
+ext {
+    tngRepoId = ""
+}
+
 def rootUrl =  {
-    if (!project.hasProperty('tngRepoId')) {
-        throw new IllegalArgumentException(
-                'You must pass the repo id (see Sonatype Repository Manager -> comtngtech-${repoId}) as parameter, e.g. -P tngRepoId=1162')
-    }
-    "https://oss.sonatype.org/service/local/repositories/comtngtech-${tngRepoId}/content/com/tngtech/archunit"
+    "https://oss.sonatype.org/service/local/repositories/${tngRepoId}/content/com/tngtech/archunit"
```

Comment:
How comes the `comtngtech-` is obsolete now? :thinking:

Reply:
The string `comtngtech-` is part of the repo-id the plugin provides. Specifying it explicitly would double it and the URL would be wrong.